app/scripts/modules/core/src/pipeline/executions/Executions.tsx

```diff
@@ -62,25 +62,6 @@ export class Executions extends React.Component<IExecutionsProps, IExecutionsSta
     };
   }
 
-  public componentWillMount(): void {
```

Comment:
moved into `componentDidMount` - not sure why we had both `componentWillMount` and `componentDidMount`. It looks like most of this code was originally in the constructor and I moved it into this method to avoid a "no pipelines found" message during initialization.